applications/crossbar/doc/user_authentication.md

```diff
@@ -12,10 +12,10 @@ Provides an auth-token via user credentials
 
 Key | Description | Type | Default | Required | Support Level
 --- | ----------- | ---- | ------- | -------- | -------------
-`account_name` | The account name of the user | `string(1..128)` |   | `false` |  
+`account_name` | The account name of the user | `string(ABC..Z)` |   | `true` |  
 `account_realm` | The account realm of the user | `string(4..253)` |   | `false` |  
-`credentials` | A hash of the uses credentials | `string(1..64)` |   | `true` |  
-`method` | The hash method | `string('md5' | 'sha')` | `md5` | `false` |  
+`credentials` | A hash of the user credentials | `string(1..64)` |   | `true` |  
+`method` | The hash method | `string(md5..sha)` | `md5` | `true` |  
```

Comment:
`string(md5 | sha)` is how we denote that the value of `method` can be either `md5` OR `sha`, `..` indicating a range of values, `|` indicating an `OR`.

Reply:
I added the .. to keep the style the same, only reason i removed the | operator is that it doesn't show properly when previewing. In this case the OR should have been used.

Reply:
Yeah Github won't render everything exactly right. The docs site (see https://docs.2600hz.com/dev/applications/crossbar/doc/user_authentication/ for instance) uses mkdocs to render the markdown into HTML so that's the rendering we target to make work.

Reply:
I understand.